Sources/Build/BuildDescription/ModuleBuildDescription.swift

```diff
@@ -121,7 +136,7 @@ public enum ModuleBuildDescription {
 
     /// Determines the arguments needed to run `swift-symbolgraph-extract` for
     /// this module.
-    package func symbolGraphExtractArguments() throws -> [String] {
+    public func symbolGraphExtractArguments() throws -> [String] {
```

Comment:
why public?

Reply:
Because it’s a witness to a public protocol requirement now